docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,130 @@
+
+    For running widget instances, you can trigger an update event, send update data, and retrieve running widget instances detail.
+
+## Prerequisites
+
+To enable your application to use the widget control functionality:
+
+1.  To use the methods and properties of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class, include the [Tizen.Applications](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.html) namespace in your application:
+
+    ```
+    using Tizen.Applications;
+    ```
+
+2.  To get information on widget application, the application has to request permission by adding the following privilege to the  `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+<a name="create_instance"></a>
+## Creating instance of Widget Control
+
+To create an instance of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code:
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
+<a name="getting_information"></a>
+## Getting Information on Widget Applications
+
+To get information on the widget applications, follow below codes:
+
+    ```
+    string mainId = control.MainAppId;
+    string packageId = control.PackageId;
+    IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+    ```
+
+<a name="listening_events"></a>
+## Listening Widget Lifecycle Events on Widget Applications
+
+To listen to the widget lifecycle events, follow below codes:
+
+    ```
+    private static void OnCreated(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        string instanceId = args.InstanceId;
+        string widgetId = args.WidgetId;
+        /// Widget application is created
+    }
+
+    private static void OnDestroyed(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        /// Widget application is destroyed
+    }
+
+    private static void OnPaused(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        /// Widget application is paused
+    }
+
+    private static void OnResumed(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        /// Widget application is resumed
+    }
+
+    control.Created += OnCreated;
+    control.Created += OnDestroyed;
+    control.Created += OnPaused;
+    control.Created += OnResumed;
+
+    string mainId = control.MainAppId;
+    string packageId = control.PackageId;
+    IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+    ```
+
+<a name="communicating_instances"></a>
+## Communicating with Running Widget Instances
+
+To communicate with the running widget instances, follow these steps:
+
```

Comment:
Please add:
Operate on the control:

1. Create an instance of the Tizen.Applications.WidgetControl class with the ID of the widget application, as shown in the following code:
    ```
     WidgetControl control = new WidgetControl(Your Widget ID);
    ```